backend/.env.example

```
@@ -29,4 +29,7 @@ DOCKERHUB_USERNAME=your_dockerhub_username
 DOCKERHUB_TOKEN=your_dockerhub_token
 
 # NASA API
-NASA_API_KEY=your_nasa_api_key
\ No newline at end of file
+NASA_API_KEY=your_nasa_api_key
+
+# Sentry (error tracking)
+SENTRY_DSN=sentry_key
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)